test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -2256,6 +2254,26 @@
                        {"certname" "foo2" "value" "testing.com"}
                        {"certname" "foo3" "value" "testing.com"}]))))))
 
+(deftest-http-app  ^:skipped tests-for-known-issues
+  [[version endpoint] fact-contents-endpoints
+   method [:get :post]]
+
+  (testing "to_string function on integers"
+    (is (= (query-result method endpoint
+                         ["extract" [["function" "to_string" "value"]]
```

Comment:
I was able to get this query to hit a validation error from [munge-result-row](https://github.com/puppetlabs/puppetdb/blob/6.x/src/puppetlabs/puppetdb/query/fact_contents.clj#L26-L32) if I added a format string to the function call. 
```
[["function" "to_string" "value" "FM9999"]]
```
See the [functions-formatting](https://www.postgresql.org/docs/11/functions-formatting.html) Postgres docs for more info. 

By adding a `:to_string` key to the `fact-contents` [row-schemas](https://github.com/puppetlabs/puppetdb/blob/6.x/src/puppetlabs/puppetdb/query/fact_contents.clj#L10-L24) the query was able to return the result we would expect. 

For this test we'll want to add the format string in the snippet above, it will still be failing, but we can work on it from there. I also had to add a call to submit factsets before running queries in the test and changed the certname to `foo1` to match the test data. The call to add data looks like: `(populate-for-structured-tests reference-time)`

Next we'll want to make two tickets: 
1. a ticket to update the AST docs to add examples of passing in format strings when using the `to_string` function on numeric and timestamp fields. 

2. a ticket to audit all of the schemas used to munge rows for different entity types for any endpoint that allows a user to use the `to_string` function. We need to make sure validation doesn't fail because we can return a `:to_string` row that the schemas aren't expecting. 

If you're up for it and wouldn't mind trying to write the tickets I can help fill in detail tomorrow if needed. We can then add a comment above this skipped test with the ticket number for 2 above and update it when we do the work.